java/test/org/openqa/selenium/grid/data/DefaultSlotMatcherTest.java

```diff
@@ -433,10 +436,10 @@ void vendorExtensionPrefixedCapabilitiesAreIgnoredForMatching() {
             "84",
             CapabilityType.PLATFORM_NAME,
             Platform.WINDOWS,
-            "goog:cheese",
-            "amsterdam",
-            "ms:fruit",
-            "mango");
+            "food:dairy",
```

Comment:
This test has been updated to verify the revised behavior, in which all extension capabilities with complex values are ignored for purposes of node matching.

Reply:
I restored the original test, altering the method name and inverting the expectation to correspond with the revised behavior.